src/tiled/brokenlinks.cpp

```diff
@@ -22,6 +22,7 @@
 
 #include "changetileimagesource.h"
 #include "documentmanager.h"
+#include "fileformat.h"
```

Comment:
Include not needed.

Reply:
Same as above, please confirm and I'll remove.